### PR TITLE
cli: --cputune: add emulatorpin.cpuset config

### PR DESCRIPTION
**Pull request by Antagonym**
First attempt at implementing new options. Hope I did it right. Got a few more to add in the near future if this pans out.

### CODE REVIEW

File: virtinst/domain/cputune.py [hunk 1]
```diff
@@ -16,6 +16,14 @@ class _VCPUPin(XMLBuilder):
...
     cpuset = XMLProperty("./@cpuset")
 
+class _EmulatorPin(XMLBuilder):
+    """
+    Class for generating <cputune> child <emulatorpin> XML
+    """
+    XML_NAME = "emulatorpin"
+
+    cpuset = XMLProperty("./@cpuset")
+
```
Comment (crobinso) [inline virtinst/domain/cputune.py:27]: Whitespace issue here: previously the classes were separated by two blanklines, but now it's 1 line before _EmulatorPin and 3 lines after.

File: virtinst/domain/cputune.py [hunk 2]
```diff
@@ -80,9 +88,11 @@ class DomainCputune(XMLBuilder):
     Class for generating <cpu> XML
     """
     XML_NAME = "cputune"
-    _XML_PROP_ORDER = ["vcpus", "cachetune", "memorytune", "vcpusched"]
+    _XML_PROP_ORDER = ["vcpus", "emulatorpin", "cachetune",
+            "memorytune", "vcpusched"]
 
     vcpus = XMLChildProperty(_VCPUPin)
+    emulatorpin = XMLChildProperty(_EmulatorPin, is_single=True)
```
Comment (crobinso) [inline virtinst/domain/cputune.py:95]: This is completely valid, but for this case it's a bit verbose. `<emulatorpin>` only has one child attribute, and can only be specified once, so it's less code to do:

```
emulatorpin_cpuset = XMLProperty("./emulatorpin/@cpuset")
```

the cli name should stay as `--cputune emulatorpin.cpuset` though. If `<emulatorpin>` ever grows more suboptions, we can split out an _EmulatorPin class like this to organize the code later and it won't affect the command line interface.

File: virtinst/domain/cputune.py (hunk 1 above)
Comment (crobinso) [reply virtinst/domain/cputune.py:27]: FWIW `./setup.py pylint` pep8 call would have caught this too

**Comment by crobinso** [comment]
Looks pretty good! One whitespace change but I recommended simplifying it a bit too. 

Any future help filling out the command line is definitely appreciated! There's a wiki page here I update periodically with missing cli options: https://github.com/virt-manager/virt-manager/wiki/Unimplemented-domain-XML-command-line-arguments

File: virtinst/domain/cputune.py (hunk 2 above)
Comment (Antagonym) [reply virtinst/domain/cputune.py:95]: Good to know. I've seen both styles around the rest of the code and I've generally tried following the style what was already there so far, but if that's the preference for "simple" suboptions, I'll stick to that unless there's a lot of suboptions.

File: virtinst/domain/cputune.py (hunk 1 above)
Comment (Antagonym) [reply virtinst/domain/cputune.py]: It did. I just derped and forgot to run it *before* committing ;)

**Comment by Antagonym** [comment]
I've simplified the implementation as suggested, which also neatly eliminates the whitespace issue at the same time.

File: virtinst/domain/cputune.py (hunk 2 above)
Comment (Antagonym) [reply virtinst/domain/cputune.py]: Next up will be iothreadpin[0-9]*. I've already implemented in a branch of my fork as an XMLChildProperty. Considering, it also only has 2 suboptions (`iothread`, and `cpuset), should I simplify that too before opening a pull request?

**Comment by crobinso** [comment]
Thanks! I squashed the two patches together and pushed.

**Pull request closed without merging**